src/Aks/readme.md

```diff
@@ -0,0 +1,206 @@
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.6.0 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.Aks`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+---
+## Generation Requirements
+Use of the beta version of `autorest.powershell` generator requires the following:
+- [NodeJS LTS](https://nodejs.org) (10.15.x LTS preferred)
+  - **Note**: It *will not work* with Node < 10.x. Using 11.x builds may cause issues as they may introduce instability or breaking changes.
+> If you want an easy way to install and update Node, [NVS - Node Version Switcher](../nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](../nodejs/installing-via-nvm.md) is recommended.
+- [AutoRest](https://aka.ms/autorest) v3 beta <br>`npm install -g autorest@beta`<br>&nbsp;
+- PowerShell 6.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g pwsh`<br>&nbsp;
+- .NET Core SDK 2.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g dotnet-sdk-2.2`<br>&nbsp;
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+> Directives
+``` yaml
+metadata:
+  authors: Microsoft Corporation
+  owners: Microsoft Corporation
+  description: 'Microsoft Azure PowerShell: $(service-name) cmdlets'
+  copyright: Microsoft Corporation. All rights reserved.
+  tags: Azure ResourceManager ARM PSModule $(service-name)
+  companyName: Microsoft Corporation
+  requireLicenseAcceptance: true
+  licenseUri: https://aka.ms/azps-license
+  projectUri: https://github.com/Azure/azure-powershell
+
+directive:
+  - where:
+      subject: Operation
+    hide: true
+  - where:
+      parameter-name: SubscriptionId
+    set:
+      default:
+        script: '(Get-AzContext).Subscription.Id'
+```
+
+``` yaml
+directive:
+  - from: managedClusters.json
+    where: $.definitions.SubResource.properties
+    transform: >
+      return {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "restype": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        }
+      }
+```
+
+### General settings
+> Values
+
+``` yaml
+service-name: Aks
+powershell: true
+azure: true
+branch: master
+repo: https://github.com/Azure/azure-rest-api-specs/blob/$(branch)
+prefix: Az
```

Comment:
I think all the metadata shoudl go in one single section, at the top

Reply:
will do it.